### DISCUSSION TITLE
Add an API endpoint for image/png

### DISCUSSION THREAD

**Discussion by iamansuman** [anuraghazra/github-readme-stats#1174] [Ideas]
I'm making a Telegram bot where a user queries for a GitHub user and the bot returns the stat.
I found this repo just right for my project but when I checked the HTTP response, the mime type was `image/svg+xml; charset=utf-8`
![image](https://user-images.githubusercontent.com/48406323/124412762-de41e900-dd6c-11eb-8b18-fbc79938477a.png)

`telebot.TeleBot().send_photo()` does not supports svg

It will be good to add an endpoint for image/png mime type.
Although it will be static with no animations, it can be universally accessed.

**Comment by rickstaa** [comment]
@iamansuthecoda a feature request for this can be found in https://github.com/anuraghazra/github-readme-stats/issues/624.
